src/r-exasol/ConnectionContext.cpp

```diff
@@ -0,0 +1,90 @@
+//
+// Created by thomas on 27/07/2021.
+//
+
+#include <r-exasol/ConnectionContext.h>
+
+
+#include <r-exasol/connection/ConnectionFactoryImpl.h>
+#include <r-exasol/rconnection/RConnection.h>
+#include <r-exasol/connection/ConnectionController.h>
+
+#include <r-exasol/rconnection/RReaderConnection.h>
+#include <r-exasol/rconnection/RWriterConnection.h>
+#include <r-exasol/odbc/OdbcSessionInfoImpl.h>
+
+namespace exa {
+    void onError(std::string e) {
+        ::error(e.c_str());
+    }
+}
+
+int exa::ConnectionContext::initConnection(const char *host, int port) {
+    destroyConnection(1);
```

Comment:
closeFD is 1 this looks scary

Reply:
I renamed the parameter name, it was quite confusing.